### PR TITLE
Translated images and documents into Simplified Chinese.

### PR DESCRIPTION
**Pull request by Warbin-Hub**
I have updated the files aspnetcore-developer-roadmap.zh-Hans.png, aspnetcore-developer-roadmap.zh-Hans.xmind, and modified ReadMe.zh-Hans.md to the latest 2024 version.

### CODE REVIEW

**Comment by Warbin-Hub** [comment]
I have updated the files aspnetcore-developer-roadmap.zh-Hans.png, aspnetcore-developer-roadmap.zh-Hans.xmind, and modified ReadMe.zh-Hans.md to the latest 2024 version.

**Review by Warbin-Hub — COMMENTED** [review]
I have updated the files aspnetcore-developer-roadmap.zh-Hans.png, aspnetcore-developer-roadmap.zh-Hans.xmind, and modified ReadMe.zh-Hans.md to the latest 2024 version.

**Review by MoienTajik — APPROVED** [review]
Thanks 🚀

**Review by MoienTajik — CHANGES_REQUESTED** [review]
Can you please compress the image?

File: ReadMe.zh-Hans.md
Comment (Warbin-Hub) [inline ReadMe.zh-Hans.md:1]: Added sponsor information in Simplified Chinese

File: aspnetcore-developer-roadmap.zh-Hans.png
Comment (Warbin-Hub) [inline aspnetcore-developer-roadmap.zh-Hans.png:1]: Compressed the image size.

File: ReadMe.md
Comment (Warbin-Hub) [inline ReadMe.md:1]: The ReadMe.md file was accidentally touched with a space, and I will restore it later.

**Review by Warbin-Hub — COMMENTED** [review]
Added sponsor information in Simplified Chinese and compressed the image size. The ReadMe.md file was accidentally touched with a space, and I will restore it later.

**Comment by Warbin-Hub** [comment]
> Can you please compress the image?

I have completed the modifications and will continue to learn along the way. If I find any omissions later, I will make additions and corrections. Thank you for sharing.

**Comment by MoienTajik** [comment]
Thanks for contribution @Warbin-Hub! 🚀

**Pull request merged**